lib/fob/page_break.ex

```diff
@@ -22,4 +28,52 @@ defmodule Fob.PageBreak do
 
     %__MODULE__{page_break | table: order.table, direction: order.direction}
   end
+
+  @doc since: "0.2.0"
+  def compare(a, b, query) when is_list(a) and is_list(b) do
+    compare(add_query_info(a, query), add_query_info(b, query))
+  end
+
+  def compare(a, b)
+
+  def compare(a, b) when is_list(a) and is_list(b) and length(a) == length(b) do
+    Enum.zip(a, b)
+    |> Enum.map(fn {a, b} -> compare(a, b) end)
+    |> Enum.find(:eq, fn comparison -> comparison != :eq end)
```

Comment:
`Enum.find/3` works nicely as a kind of circuit-breaker here

it tries to find the first comparison which is not equal. if it can't find any, `a == b`!